server/src/workspace/workspace-schema-builder/utils/get-resolver-args.util.ts

```diff
@@ -75,6 +75,24 @@ export const getResolverArgs = (
           isNullable: false,
         },
       };
+    case 'updateMany':
+      return {
+        data: {
+          kind: InputTypeDefinitionKind.Update,
+          isNullable: false,
```

Comment:
As it's an `updateMany` we must add `isArray` option.

```suggestion
          isNullable: false,
          isArray: true,
```

Reply:
Please refer to my comment above